hasktorch/src/Torch/Script.hs

```diff
@@ -317,52 +388,38 @@ printGraph = cast1 LibTorch.printGraph
 -- On the other hand, torch.onnx.export of python works.
 -- onnx's symbol map is in python code.
 -- https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_opset9.py
--- 
+--
 -- If you need onnx-file, at first make torchscript by trace , then convert torchscript into onnx by python-code.
 printOnnx :: Graph -> IO String
 printOnnx = cast1 LibTorch.printOnnx
 
 graphToJitGraph :: Graph -> IO JitGraph
-graphToJitGraph (UnsafeGraph graph) = do
+graphToJitGraph (UnsafeGraph graph) =
```

Comment:
twitter has already congratulated you on this one, but I'd like to say it as well: Thank you, well done!